_posts/2015-01-06-airportquiz.md

```diff
@@ -0,0 +1,10 @@
+---
+category: app
+title: "AirportQuiz"
+icon: "/assets/AirportQuiz.svg"
+app_store_link: "https://itunes.apple.com/us/app/airport-quiz/id898891728?ls=1&mt=8"
+---
+
+## What can I say? I am ruling it.
+
+@daehn, #1 in all AirportQuiz leaderboards
```

Comment:
@daehn are you ok with your mention?
![daehn_quote](https://cloud.githubusercontent.com/assets/759730/7378061/0592734e-edeb-11e4-971b-f318d9ca3130.png)